wagtailcache/cache.py

```diff
@@ -163,7 +163,18 @@ def process_response(self, request: WSGIRequest, response: HttpResponse) -> Http
             timeout = self._wagcache.default_timeout
         patch_response_headers(response, timeout)
         if timeout:
+            # Create caches key dict.
+            if not self._wagcache.get('keyring'):
+                self._wagcache.set('keyring', {})
+
             cache_key = learn_cache_key(request, response, timeout, None, cache=self._wagcache)
+
+            # Track cache keys.
+            uri = clean_uri(request.build_absolute_uri())
+            keyring = self._wagcache.get('keyring')
```

Comment:
This could provide the default fallback value, and then lines 166-169 above would not be necessary. This way it is not looking up 'keyring' twice, each of which is a performance hit. E.g:
```python
keyring = self._wagcache.get('keyring', {})
```